src/django_components/slots.py

```diff
@@ -150,34 +147,132 @@ class ComponentSlotContext:
 
 
 class SlotNode(BaseNode):
-    """Node corresponding to `{% slot %}`"""
-
-    def __init__(
-        self,
-        nodelist: NodeList,
-        params: TagParams,
-        trace_id: str,
-        node_id: Optional[str] = None,
-        is_required: bool = False,
-        is_default: bool = False,
-    ):
-        super().__init__(nodelist=nodelist, params=params, node_id=node_id)
-
-        self.is_required = is_required
-        self.is_default = is_default
-        self.trace_id = trace_id
+    """
+    Slot tag marks a place inside a component where content can be inserted
+    from outside.
 
-    @property
-    def active_flags(self) -> List[str]:
```

Comment:
`active_flags` was also moved to `BaseNode`, so now all our Node classes have this property